### DISCUSSION TITLE
What can be the reson of openConnectModal is undefined?

### DISCUSSION THREAD

**Discussion by azzztec** [rainbow-me/rainbowkit#1721] [Q&A] [answered]
One user has problem with connection wallet. I found that he has openConnectModal is undefined, but only that user has that problem. What can be the reason?

**Comment by 0xmkh** [comment, marked as answer]
@azzztec The only time `openConnectModal` will be `undefined` is if your wallet is connected. 

Another case you might have `undefined` is if you're using our [custom authentication](https://www.rainbowkit.com/docs/custom-authentication) and you `status` to either `unauthenticated` or `loading`.
